index.html

```diff
@@ -27,63 +27,78 @@
 
             <div class="navbar-mobile nav-container">
                 <a href="http://#" class="nav-text">Welcome</a>
-                <div class="hamburgur-menu">
+                <div class="hamburgur-menu" id="menu-icon">
                     <i class="fa-solid fa-bars"></i>
                 </div>
+                <div id="mobile-menu" class="hide-menu">
+                    <div id="mobile-menu-container">
+                        <div class="close-menu">
+                            <svg width="18" height="18" viewBox="0 0 14 14" fill="none"
+                                xmlns="http://www.w3.org/2000/svg">
+                                <path fill-rule="evenodd" clip-rule="evenodd"
+                                    d="M0.292893 0.292893C0.683417 -0.0976311 1.31658 -0.0976311 1.70711 0.292893L7 5.58579L12.2929 0.292893C12.6834 -0.0976311 13.3166 -0.0976311 13.7071 0.292893C14.0976 0.683417 14.0976 1.31658 13.7071 1.70711L8.41421 7L13.7071 12.2929C14.0976 12.6834 14.0976 13.3166 13.7071 13.7071C13.3166 14.0976 12.6834 14.0976 12.2929 13.7071L7 8.41421L1.70711 13.7071C1.31658 14.0976 0.683417 14.0976 0.292893 13.7071C-0.0976311 13.3166 -0.0976311 12.6834 0.292893 12.2929L5.58579 7L0.292893 1.70711C-0.0976311 1.31658 -0.0976311 0.683417 0.292893 0.292893Z"
+                                    fill="#93A79D" />
+                            </svg>
+                        </div>
+                        <ul id="mobile-links">
+                            <li><a href="#portfolio">Portfolio</a></li>
+                            <li><a href="#about">About</a></li>
+                            <li><a href="#contact-form">Contact</a></li>
+                        </ul>
+                    </div>
+                </div>
```

Comment:
- [ ] Kindly make sure that the user cannot scroll in the page itself when the mobile menu is open. Currently, if you have the mobile menu open, you can scroll past it onto the page content.